File: ExamenUnidad2__JAVA/ExamenUnidad2/src/examenunidad2/Persona.java

package examenunidad2;

public class Persona 
{
    
    public static int IDENTIFICADOR = 0;
    private String name;
    private String apellidos;
    protected String fechaDeNacimiento;
    private String CURP;
    private int id;
  

    public Persona(String name, String apellidos, String fechaDeNacimiento, String CURP)
    {
        this.name = name;
        this.apellidos = apellidos;
        this.fechaDeNacimiento = fechaDeNacimiento;
        this.CURP = CURP;
        
    }

    public Persona() 
    {
    }
   
    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getApellidos() {
        return apellidos;
    }

    public void setApellidos(String apellidos) {
        this.apellidos = apellidos;
    }

    public String getFechaDeNacimiento() {
        return fechaDeNacimiento;
    }

    public void setFechaDeNacimiento(String fechaDeNacimiento) {
        this.fechaDeNacimiento = fechaDeNacimiento;
    }

    public String getCURP() {
        return CURP;
    }

    public void setCURP(String CURP) {
        this.CURP = CURP;
    }

    public static int getIDENTIFICADOR() {
        return IDENTIFICADOR;
    }

    public static void setIDENTIFICADOR(int IDENTIFICADOR) {
        Persona.IDENTIFICADOR = IDENTIFICADOR;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }
 
}

File: ExamenUnidad2_PYTHON 1/ExamenUnidad2_PYTHON 1/ExamenUnidad2/Persona.py
class Persona:
    IDENTIFICADOR = 0

    def __init__(self, name, apellidos, fecha_de_nacimiento, CURP):
        self.name = name
        self.apellidos = apellidos
        self.fecha_de_nacimiento = fecha_de_nacimiento
        self.CURP = CURP
        self.id = Persona.IDENTIFICADOR
        Persona.IDENTIFICADOR += 1

    def get_name(self):
        return self.name

    def set_name(self, name):
        self.name = name

    def get_apellidos(self):
        return self.apellidos

    def set_apellidos(self, apellidos):
        self.apellidos = apellidos

    def get_fecha_de_nacimiento(self):
        return self.fecha_de_nacimiento

    def set_fecha_de_nacimiento(self, fecha_de_nacimiento):
        self.fecha_de_nacimiento = fecha_de_nacimiento

    def get_CURP(self):
        return self.CURP

    def set_CURP(self, CURP):
        self.CURP = CURP

    @staticmethod
    def get_IDENTIFICADOR():
        return Persona.IDENTIFICADOR

    @staticmethod
    def set_IDENTIFICADOR(IDENTIFICADOR):
        Persona.IDENTIFICADOR = IDENTIFICADOR

    def get_id(self):
        return self.id

    def set_id(self, id):
        self.id = id

